.testing/Makefile

```diff
@@ -76,6 +76,7 @@ export CC
 export MPICC
 export FC
 export MPIFC
+export FMS_COMMIT
```

Comment:
Was this meant to be part of the patch?

Reply:
Yeah, helps to test FMS2.

> I have also added an export statement to the .testing Makefile to allow
for configuration of the FMS branch within the CI test suite.

I guess it could have been a separate commit. :shrug: